.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,20 @@
+name: Update dependencies
+
+on: [repository_dispatch]
+
+jobs:
+  update-webex-component-adapter-interfaces:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'component-adapter-interfaces:version-changed'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          npm install @webex/component-adapter-interfaces --save
+          if [ -n "$(git status --porcelain)" ]; then
+            git config user.name "GitHub Actions Bot"
+            git config user.email "<>"
+            git add .
+            git commit -m 'fix(@webex/component-adapter-interfaces): update to new version'
```

Comment:
@lalli-flores I would prefer dependency updates to use `build()`, as they should not yet use the features in the new dependency version (eg: a new field in an adapter interface). It's the subsequent commits that start using those features, and it's those commits that should generate new versions on the current repository.

Reply:
I would prefer the same commit message we are currently using:
`build(package): update component adapter interfaces to v1.2.3`

Reply:
yeah, I'm good with your commit message suggstion